Add outdated_translation variable

huey735 opened the pull request:
On @m52go's advice I changed the construction of the outdated
translation alert tool to be spefic to the page instead of
the language. I added a "outdated_translation" variable to
the front matter of the relevant translated pages. They'll
have the values 'true' or 'false' according to the state.

huey735 committed:
Add outdated_translation variable

On @m52go's advice I changed the construction of the outdated
translation alert tool to be spefic to the page instead of
the language. I added a "outdated_translation" variable to
the front matter of the relevant translated pages. They'll
have the values 'true' or 'false' according to the state.

erciccione commented on _includes/main_nav_tr.html:
```diff
@@ -1,6 +1,13 @@
+
+     <!-- Outdated translation alert tool tag-->
+     {% if page.outdated_translation %}
+       <span class="navbar-text" style="color: orange;">Outdated translation</span>
```
I suggest to add a link toTransifex, so that potential contributors know where to go if they want to help translate the page. The message could be something like `Outdated translation - [Help us to improve it!](https://www.transifex.com/bisq/bisq-website/)`.

Would be also cool to have the phrase translatable imho, so that when somebody is using another language the warning will be in their language.

huey735 replied:
@erciccione what do you mean by message?

erciccione replied:
The content of the warning (that now is only `Outdated translation`)

huey735 replied:
Well, for that I think we'll have to move it from the navbar. There's not enough space there as it is.

erciccione replied:
I don't think the navbar is the best place for that warning. I would put it in a dedicated snippet under the navbar. That way would be easier to see for the user, we would have more space so we can be more descriptive and styling form mobile would be easier to manage.

huey735 replied:
@erciccione @m52go I've moved it from the nav bar to the page body. Maybe we can work more on the text to display something more inviting and to make sure they check out the English version.
![outdatedtool](https://user-images.githubusercontent.com/44140017/64205787-b5aae880-ce90-11e9-9b9e-173bac191ffe.png)

huey735 replied:
@erciccione how about this, following the standard warning sign palette. Orange text on green background doesn't stand out that much.
![outdated2](https://user-images.githubusercontent.com/44140017/64241528-d6f4ee80-cefb-11e9-91a7-7874cf5c9671.png)

huey735 replied:
The black text however doesn't imply that there's an hyperlink to be clicked.

erciccione replied:
I'm not great with colours so, keep that in mind :P, but i personally think this solution is better than simple text. What about having the hyperlink underlined? or underlined on hover?

huey735 replied:
@ericcione now it looks like this when hovered
![outdated3](https://user-images.githubusercontent.com/44140017/64245588-18d56300-cf03-11e9-8dff-9fe630083679.png)

erciccione replied:
I guess could be fine. Better wait @m52go's input on this

huey735 committed:
Deplace outdated_translation variable

Move outdated_translation tool from _includes/main_nav_tr.html
to _layouts/page.html.

huey735 committed:
Make warning text more visible

Changed the text color to black and background-color to orange.
And on hover the text-decoration is underline and the color is
blue.

huey735 committed:
Make warning text fit the website theme

m52go commented on _includes/main_nav_tr.html:
```diff
@@ -1,6 +1,8 @@
  <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top pl-md-5 pr-md-5 pl-sm-3 pr-0">
 
-    <a class="navbar-brand" href="{{ site.url }}/{{ page.lang }}/"><img src="{{ site.url }}/images/bisq-logo.svg" height="30"/></a>
+   <div>
```
Any particular reason for adding this div element?

huey735 replied:
@m52go They were vestiges of when the warning tool was placed in the navbar.

m52go reviewed (commented):
This looks ready to go, thanks! Just one small question (see comment).

huey735 committed:
Remove unnecessary div

wiz commented on _layouts/page.html:
```diff
@@ -4,6 +4,13 @@
+      <!-- Outdated translation alert tool tag-->
+      <div class="col-md-12 order-md-1 text-md-left">
+        {% if page.outdated_translation %}
+          <p class="text-muted">⚠ This page contains some outdated translations. Help us improve <a href="https://www.transifex.com/bisq/bisq-website/">here</a>!</p>
```
Don't make links like this:
<a href="/foo">Click here</a> for more information</a>
Make links like this:
<a href="/foo">More information</a>
